python/wrapper_tests/test_01_synthetic_collection.py

```diff
@@ -0,0 +1,128 @@
+# This example generates a synthetic in-memory collection with 100 items and 60 tokens.
+# First 40 tokens are split on 10 clear topics according to this condition: "(token_id % 10) == (item_id % 10)"
+# Last 20 tokens are randomly allocated to documents.
+# The example demonstrates that in this clear situation BigARTM can precisely recover all topics.
+
+import artm.wrapper
+import artm.wrapper.messages_pb2 as messages
+import artm.wrapper.constants as constants
+import random
+import uuid
+import shutil
+import os
+import pytest
+
+def test_func():
+    # Set some constants
+    num_tokens = 60
+    num_items = 100
+    batches_folder = os.path.join(os.getcwd() + '/temp_batches')
+    pwt = 'pwt'
+    nwt = 'nwt'
+
+    # Generate small collection of random items
+    batch = messages.Batch()
+    batch.id = str(uuid.uuid4())
+    for token_id in xrange(num_tokens):
+        batch.token.append('token' + str(token_id))
+
+    for item_id in xrange(num_items):
+        item = batch.item.add()
+        item.id = item_id
+        field = item.field.add()
+        for token_id in range(0, num_tokens):
+            field.token_id.append(token_id)
+            background_count = random.randint(1, 5) if (token_id >= 40) else 0
+            topical_count = 10 if (token_id < 40) and ((token_id % 10) == (item_id % 10)) else 0
```

Comment:
denote the number of those topics with a correct word: 
meaningful_, reasonable_topics, target_topics, ...